packages/vats/test/bootstrapTests/test-vaults-integration.js

```diff
@@ -69,6 +69,110 @@ test.after.always(t => {
   t.context.shutdown && t.context.shutdown();
 });
 
+/** @type {<X>(a: X[], b: X[]) => X[]} */
+const setDiff = (a, b) => a.filter(x => !b.includes(x));
+
+test('audit bootstrap exports', async t => {
+  const expectedIfaces = [
+    // in bridgeCoreEval()
+    // TODO(#7576): support unregister
+    'coreHandler',
+    // in bridgeProvisioner()
+    // TODO(#7576): support unregister
+    'provisioningHandler',
+    // makePrioritySendersManager() TODO(#7576): support unregister
+    'prioritySenders manager',
+    // TODO? move to provisioning vat?
+    'clientCreator',
+    // in registerNetworkProtocols(). TODO: outboard #7044
+    'ProtocolHandler', // in makeLoopbackProtocolHandler()
+    'callbacks',
+    'listener',
+    // TODO(#7563): disable stakeFactory in test-vaults-config
+    'stakeReporter',
+    // in startWalletFactory()
+    // TODO(#5885): vbank should provide a facet attenuated
+    // to only provide getAssetSubscription
+    // meanwhile, expose the whole poolBank rather than
+    // export this Far object from bootstrap?
+    'AssetPublisher',
+    // in addBankAssets()
+    // XXX is attenuation needed here?
+    'AssetHub',
+    // XXX price-feed-proposal uses makeIssuerKit
+    'USD brand',
+    'ATOM brand',
+    // from makeAddressNameHubs(),
+    // makeMyAddressNameAdminKit(),
+    // makeAgoricNamesAccess()
+    'nameHub',
+    'nameAdmin',
+  ];
+
+  const { controller } = t.context;
+  const kState = controller.dump();
+  const bootstrapExports = kState.kernelTable.filter(
+    o => o[1] === 'v1' && o[2].startsWith('o+'),
+  );
```

Comment:
Just thought about something. Should we be worried about bootstrap having some promises to which it remains the decider?

Reply:
well, now I am worried. Is it straightforward to find them in the kernel dump?

Reply:
good news: there are none!